examples/build.gradle

```diff
@@ -15,7 +15,7 @@ subprojects {
     version = "0.1.0-SNAPSHOT"
 
     ext {
-        openTelemetryVersion = "1.0.0"
+        openTelemetryVersion = "1.1.0-SNAPSHOT"
```

Comment:
I thought about this in the past, downside is that users will copy paste and we probably want to use the released version there.

One option is to make this configurable, so we keep it here as the stable release but when building for testing we force the snapshot version.

Reply:
I think I know how to make this parameterized, and have the CI jobs run against the snapshot, but the examples themselves use the stable release. Thanks for the suggestion!

Reply:
ok, check it out. defined these versions as project properties, so can easily be different between CI and copy/pasting for examples.